### PR TITLE
chore: rewrite README.md for development  and add `data/**/*.jsonl` to .gitignore

### PR DESCRIPTION
**Pull request by Kout95**
### What
- Rewrite README.md to detail how to install project for development.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,49 +46,84 @@ You will then need to import from a JSONL dump (see instructions below).
...
-make up
-```
+##### Direnv
+You can follow our tutorial to install [direnv](https://openfoodfacts.github.io/openfoodfacts-server/dev/how-to-use-direnv/)
 
-However, this tends to be slower than developing locally.
+Add a `.envrc` file at the root of the project with the following content:
+```shell
+export USER_GID=1001
+export USER_UID=1001
```
Comment (Marc-AntoineA) [inline README.md:60]: On my computer it’s 1000, should match `id -u`

File: README.md [hunk 2]
```diff
@@ -46,49 +46,84 @@ You will then need to import from a JSONL dump (see instructions below).
 
 ### Development
 
-For development, you have two options for running the service:
-1. Docker
-2. Locally
```
Comment (Marc-AntoineA) [inline README.md:51]: `locally` installation is still possible

File: README.md [hunk 3]
```diff
@@ -46,49 +46,84 @@ You will then need to import from a JSONL dump (see instructions below).
...
+export USER_GID=1001
+export USER_UID=1001
 
-To develop locally, create a venv, install dependencies, then run the service:
-```console
-virtualenv .
-source venv/bin/activate
-pip install -r requirements.txt
-uvicorn app.api:app --reload --port=8001 --workers=4
+export CONFIG_PATH=data/config/openfoodfacts.yml
```
Comment (Marc-AntoineA) [inline README.md:62]: Specific for openfoodfacts

**Review by Marc-AntoineA — COMMENTED** [review]
I think you should keep the `locally` option.

File: README.md [hunk 4]
```diff
@@ -46,49 +46,84 @@ You will then need to import from a JSONL dump (see instructions below).
...
+##### Docker
+- First of all, you need to have Docker installed on your machine. You can download it [here](https://www.docker.com/products/docker-desktop).
+- Be sure you can [run docker without sudo](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user)
 
-To develop on docker, make the changes you need, then build the image and compose by running:
-```console
-make up
-```
+##### Direnv
+You can follow our tutorial to install [direnv](https://openfoodfacts.github.io/openfoodfacts-server/dev/how-to-use-direnv/)
```
[the remark this answers is not in the document]
Comment (alexgarel) [reply README.md]: FYI @Kout95: you applied it wrongly: if we have a `[^winEnvrc]` reference, we need the `[^winEnvrc]: ...` note.

[the remark this answers is not in the document]
Comment (alexgarel) [reply README.md]: fixed in 0a3ec566cd8257cba4448c13f6707eb93120361d

**Pull request merged**